mm2src/mm2_core/src/mm_ctx.rs

```diff
@@ -203,6 +201,8 @@ impl MmCtx {
         self.rmd160.or(&|| &*DEFAULT)
     }
 
+    pub fn rmd160_hex(&self) -> String { hex::encode(self.rmd160().as_slice()) }
+
     pub fn shared_db_id(&self) -> &H160 {
```

Comment:
no need, it will be a duplicate of `MmCtx::rmd160()`

Reply:
I guess we cannot just rename rmd160() to default_db_id() as it may be used not only for the db use case.
Having default_db_id() function would make it easier to understand the sql connection pool code.
Also I'd suggest renaming MmCtx::shared_db_id() to default_shared_db_id() because there is another shared_db_id() in MmCoin with different semantics.

Reply:
make sense done thanks.